Extensions/PlatformBehavior/PlatformerObjectBehavior.cpp

```diff
@@ -65,6 +66,8 @@ PlatformerObjectBehavior::GetProperties(
             "jumps."));
   properties[_("Max. falling speed")].SetValue(
       gd::String::From(behaviorContent.GetDoubleAttribute("maxFallingSpeed")));
+  properties[_("Ladder Climbing speed")].SetValue(
```

Comment:
Missing S on speed.  Use the same name as on line 122

Reply:
Thanks a lot, @Bouh; But on the contrary, I think https://github.com/4ian/GDevelop/pull/1578/files?file-filters%5B%5D=.cpp#r436284724.

Reply:
There is still a weird casing, it's either "Ladder Climbing Speed" (all capitals, i.e: title case) or not: "Ladder climbing speed" for the property name... but not a capital to one word and not to the other :)

Let's be consistent with "Max. falling speed" and use "Ladder climbing speed" here and below.